src/components/Identicon/index.tsx

```diff
@@ -18,8 +17,7 @@ const StyledAvatar = styled.img`
   border-radius: inherit;
 `
 
-export default function Identicon({ size }: { size?: number }) {
-  const { account } = useWeb3React()
+export default function Identicon({ account, size }: { account: string; size?: number }) {
```

Comment:
why?

Reply:
We must pass the account in order for the memoized account to be available when rendering during a chain switch, because technically the account is not connected during that switch, so `useWeb3React()` will not return one.

See https://github.com/Uniswap/interface/pull/6662/files#diff-72207ac383212f07b07e35204f4ee12dcee52d563af7ed7f4e3c669a6544a57dR137, where the account is memoized using `useLast` in case a chain switch occurs.